src/commands/delete.js

```diff
@@ -0,0 +1,53 @@
+const utils = require('../utils');
+
+const _delete = (context, appOrVersion, version) => {
+  if (appOrVersion === 'version' && !version) {
+    const message = 'Error: No version - provide a version like "1.0.0"...';
+    return Promise.reject(new Error(message));
+  }
+  return utils.checkCredentials()
+    .then(() => utils.getLinkedApp())
+    .then((app) => {
+      const deletePrepMessage = (appOrVersion === 'version')
```

Comment:
Indeed that's not a pattern we have here, but in a future refactor it would make things clearer indeed.